client/src/components/Admin/IssueForm/index.js

```diff
@@ -101,13 +101,19 @@ class IssueForm extends React.Component {
           hidden={!showActiveIssueWarning}
         >Det er allerede en aktiv sak!</p>
         <label className="IssueForm-textarea">
-          <div className="IssueForm-label">Beskrivelse</div>
+          <h4 className="IssueForm-label">Beskrivelse av saken</h4>
```

Comment:
4 seems high. I don't have the DOM in front of me, but I think this should be h2 maybe h3?